util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -329,12 +341,17 @@ protected static void checkDates(
     List<Integer> dateLenList = new ArrayList<>(dateLen.keySet());
     if (dateLenList.size() > 1) {
       StatValidationEntry.Builder inconsistentDateCounter = StatValidationEntry.newBuilder();
-      inconsistentDateCounter.setCounterKey("StatsCheck_Inconsistent_Date_Granularity");
+      String inconsistentDateCounterKey = "StatsCheck_Inconsistent_Date_Granularity";
+      inconsistentDateCounter.setCounterKey(inconsistentDateCounterKey);
       // When there are multiple date granularity, the problem points will be those that are of a
       // different date granularity than the most common one.
       dateLenList.sort((d1, d2) -> dateLen.get(d2).size() - dateLen.get(d1).size());
       for (int i = 1; i < dateLenList.size(); i++) {
         dateLen.get(dateLenList.get(i)).forEach(inconsistentDateCounter::addProblemPoints);
+        logCtx.incrementCounterBy(
+            Level.LEVEL_WARNING.name(),
+            inconsistentDateCounterKey,
+            dateLen.get(dateLenList.get(i)).size());
```

Comment:
Wondering why we don't just update this counter once for the series? Add comments?

Reply:
I thought it would make sense for the counter number to match the number of problem points for this counter. added comment.

Reply:
My feeling is it is more useful to see the number of series' where a problem occurs, than the points in a single series?

Reply:
that makes sense, made the change